scripts/build-and-test

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+
+set -e
+
+run_tests=true
+build_jobs=1
+
+script_path=$(dirname $(which $0))
+dawn_source_dir=$script_path/..
+dawn_build_dir=$dawn_source_dir/build
+dawn_install_dir=/usr/local
+
+# Build arguments
+build_args=()
+
+while [ "$1" != "" ]; do
+    case $1 in
+    -j | --parallel)
+        build_jobs=$2
+        shift
+        ;;
+    -b | --dawn-build-dir)
+        dawn_build_dir=$2
+        shift
+        ;;
+    -c | --config)
+        build_type=$2
+        shift
+        ;;
+    --clang-gridtools-source-dir)
+        cg_source_dir=$2
+        shift
+        ;;
+    --clang-gridtools-build-dir)
+        cg_build_dir=$2
+        shift
+        ;;
+    -d | --dawn-source-dir)
+        dawn_source_dir=$2
+        shift
+        ;;
+    -n | --no-tests)
+        run_tests=false
+        ;;
+    -i | --dawn-install-dir)
+        dawn_install_dir=$2
+        shift
+        ;;
+    *)
+        build_args+=($1)
+        ;;
+    esac
+    shift
+done
+
+# Dawn
+cmake -S $dawn_source_dir -B $dawn_build_dir \
+    -DBUILD_TESTING=ON \
+    -DCMAKE_INSTALL_PREFIX=$dawn_install_dir \
+    ${build_args[@]}
+
+time cmake --build $dawn_build_dir --config $build_type --parallel $build_jobs --target install
```

Comment:
should the timings stay?

Reply:
Probably not. I could make it an option.

Reply:
Made it an option `-t` or `--timing`.